cpu/sam0_common/periph/spi.c

```diff
@@ -399,19 +385,19 @@ void spi_init_pins(spi_t bus)
 void spi_deinit_pins(spi_t bus)
 {
     mutex_lock(&locks[bus]);
-
```

Comment:
spi_deinit_pins does not deinit pins anymore

Reply:
The pins cannot be kept attached to the SERCOM when releasing it. So they will be detached by default and only be routed to the SERCOM between `spi_acquire()` and `spi_release()`.

Just locking the mutex will block calls to `spi_acquire()`, so the pins will remain detached until the mutex is unlocked again via `spi_init_pins()`. `spi_init_pins()` will also restore the GPIO mode just in case, but not attach them to the SERCOM.